kotlinx.fuzz.CasrAdapter/CasrAdapter/Cargo.toml

```diff
@@ -0,0 +1,12 @@
+[lib]
```

Comment:
My original idea was to move CasrAdapter to a separate repository, so that it is fully independent from kotlinx.fuzz. I think we can create a repo for it in plan-maven and just migrate it there

Reply:
@FerrumBrain I created a [repo](https://github.com/plan-research/casr-adapter)